tests/common/snappi/snappi_helpers.py

```diff
@@ -308,3 +309,8 @@ def wait_for_arp(snappi_api, max_attempts=10, poll_interval_sec=1):
                   "ARP is not resolved in {} seconds".format(max_attempts * poll_interval_sec))
 
     return attempts
+
+
+class class_enable_vector(Enum):
```

Comment:
Do we really need Enum for this? bool should be enough?

Reply:
Can you model this exactly as the class vector - This is a 8 bit field. bit0-7 represent priorities 0-7. if bit 3 is enabled, it means this is a pause frame and the pause quanta is present in the time (class 3) field

Reply:
agree this should be bool. no other values are possible

Reply:
Changed to bool